onchain/src/social/account.cairo

```diff
@@ -1,35 +1,47 @@
-use starknet::{ContractAddress, get_caller_address, get_contract_address, contract_address_const};
-use super::profile::NostrProfile;
 use super::request::SocialRequest;
 use super::transfer::Transfer;
 
 
 #[starknet::interface]
 pub trait ISocialAccount<TContractState> {
     fn get_public_key(self: @TContractState) -> u256;
+}
+
+#[starknet::interface]
```

Comment:
Why do you need both `ISocialAccount` and  `ISocialAccountMixin`? Why not just `ISocialAccount` with `handle_transfer`?